legion/engine/physics/data/physics_manifold.hpp

```diff
@@ -37,6 +37,62 @@ namespace legion::physics
 
         bool isColliding;
 
+        /*void DEBUG_checkIDAndBreak(std::string firstID,std::string secondID) const
+        {
+            auto idHA = entityA.get_component_handle<identifier>();
+            auto idHB = entityB.get_component_handle<identifier>();
+
+            if (idHA && idHB)
+            {
+                auto str1 = idHA.read().id;
+                auto str2 = idHB.read().id;
+
+                if ((str1 == firstID && str2 == secondID)
+                    || (str2 == firstID && str1 == secondID))
+                {
+                    DebugBreak();
+                }
+            }
+        }*/
+
```

Comment:
can this be removed

Reply:
the reason this is commented is because the build action fails when this is added, despite the fact that it can still run, I will need it later